app/controllers/initiatives_controller.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class InitiativesController < ApplicationController
+  before_action :set_initiative, only: %i[show destroy update]
+
+  def index
+    @initiatives = Initiative.all
+  end
+
+  # POST
+  def create
+    @initiative = Initiative.new(initiative_params)
+    return if @initiative.save
+
+    render json: @initiative.errors, status: :unprocessable_entity
```

Comment:
This is not correct, you shouldn't return to go to a view. you should do:
`render json: @initiative.errors, status: :unprocessable_entity unless @initiative.save`

Reply:
Thanks! Fixed!